highLevelInterface/pwaFit.cc

```diff
@@ -97,10 +96,18 @@ rpwa::hli::pwaFit(const pwaLikelihood<complex<double> >& L,
 #endif
 	const bool         quiet                 = not verbose;
 
+	const double massBinMin    = binningMap.at("mass").first;
+	const double massBinMax    = binningMap.at("mass").second;
+
 	// report parameters
 	printInfo << "running pwaFit with the following parameters:" << endl;
-	cout << "    mass bin [" << massBinMin << ", " << massBinMax << "] GeV/c^2" << endl
-	     << "    seed for random start values ................... "  << seed                    << endl
+	cout << "    mass bin [" << massBinMin << ", " << massBinMax << "] GeV/c^2" << endl;
+	for(const auto& variable_borders: binningMap){
```

Comment:
maybe one could improve the naming: why not just call variable_borders "bin", or "binBorders", at least "variableBorders".